_field-types/mapping-parameters/analyzer.md

```diff
@@ -2,89 +2,207 @@
-          "stopwords": ["the", "a", "and", "or"]
-        },
-        "my_stemmer": {
-          "type": "stemmer",
-          "language": "english"
+          "stopwords": "_english_"
         }
       }
     }
   },
   "mappings": {
     "properties": {
-      "my_text_field": {
+      "product_name": {
         "type": "text",
-        "analyzer": "my_custom_analyzer",
-        "search_analyzer": "standard",
-        "search_quote_analyzer": "my_custom_analyzer"
+        "analyzer": "index_analyzer",
+        "search_analyzer": "search_analyzer",
+        "search_quote_analyzer": "index_analyzer"
       }
     }
   }
 }
 ```
 {% include copy-curl.html %}
 
-In this example, the `my_custom_analyzer` uses the standard tokenizer, converts all tokens to lowercase, applies a custom stop word filter, and applies an English stemmer.
+Next, add sample documents to the index:
+
+```json
+PUT /product_catalog/_doc/1
+{
+  "product_name": "The Smart Watch Pro"
+}
+```
+{% include copy-curl.html %}
+
+```json
+PUT /product_catalog/_doc/2
+{
+  "product_name": "A Smart Watch Ultra"
+}
+```
+{% include copy-curl.html %}
 
-You can then map a text field so that it uses this custom analyzer for both index and search operations:
+Search your index for the phrase "the smart watch" (in quotation marks):
 
 ```json
-"mappings": {
-  "properties": {
-    "my_text_field": {
-      "type": "text",
-      "analyzer": "my_custom_analyzer"
+GET /product_catalog/_search
+{
+  "query": {
+    "query_string": {
+      "query": "\"the smart watch\"",
+      "default_field": "product_name"
     }
   }
 }
 ```
 {% include copy-curl.html %}
+
+Because the query is enclosed in quotation marks, it becomes a phrase query. Phrase queries use the `search_quote_analyzer`, which preserves stop words. As a result, the query "the smart watch" matches only documents containing that exact phrase, so the response includes only the first document:
+
+```json
+{
+  "took": 263,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 1,
+      "relation": "eq"
+    },
+    "max_score": 0.48081374,
+    "hits": [
+      {
+        "_index": "product_catalog",
+        "_id": "1",
+        "_score": 0.48081374,
+        "_source": {
+          "product_name": "The Smart Watch Pro"
+        }
+      }
+    ]
+  }
+}
+```
+
+Now, search for the text matching "the smart watch" (without quotation marks):
+
+```json
+GET /product_catalog/_search
+{
+  "query": {
+    "query_string": {
+      "query": "the smart watch",
+      "default_field": "product_name"
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+Because the query is not enclosed in quotation marks, it is a term query. Term queries use the `search_analyzer`, which tokenizes the text and removes stop words. As a result, the query the smart watch is analyzed into the tokens `[smart, watch]` and matches both documents:
```

Comment:
```suggestion
Because the query is not enclosed in quotation marks, it is a term query. Term queries use the `search_analyzer`, which tokenizes the text and removes stop words. As a result, the query is analyzed into the tokens `[smart, watch]` and matches both documents:
```